docs/OnTrack/Deployment/Google Cloud/Overview.md

```diff
@@ -30,40 +36,121 @@ is a small, initial footprint that has a publicly facing network and accessible
+[here](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/gcp-instance-rest.json).
+
+Once we had the instance up and running, we connected to the instance using command-line shell via
+SSH. In the Google Cloud console, you can view the options to connect by clicking the drop-down menu
+beside _Connect SSH_ on the instance view.
+
+![gcp-instance-connect](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/gcp-instance-connect.png "Instance connection options")
+
+Alternatively, you can SSH to the instance via gcloud CLI -
+
+```
+gcloud compute ssh --zone "australia-southeast2-a" "instance-1"  --project "sit-22t1-ontrack-deplo-d026375"
+```
+
+There were a few packages that needed to be installed on the host, such as -
+
+- yum-utils
+- yum-config-manager (required to setup docker-ce repo, see
+  [Docker docs](https://docs.docker.com/engine/install/centos/) for Centos)
+- docker-ce, docker-ce-cli, containerd.io, docker-complose-plugin
+- git
+- openssl
+- nano (optional)
+
+Once installed using _yum_, we had the minimum requirements to get started on the Doubtfire
+deployment.
+
+### Deploying OnTrack
+
+From here, we pulled down the doubtfire-deploy repository, generated a new certificate and private
+key for the host, and ran docker compose to deploy the containers for the OnTrack deployment.
+
+![gcp-instance-docker](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/gcp-instance-docker.png "Instance running docker containers")
+
+And we can browse to OnTrack over the internet using our public IP address via HTTPS (port 443).
+
+![ontrack-login](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/ontrack-login.png "OnTrack Login")
 
 ## Next stages
 
-Since we have a functional and accessible instance of Doubtfire running in GCP, the next stage would
-be to focus on automation with CI/CD pipelines to deploy the Thoth Tech OnTrack to GCP. It would
-also be a bonus to get LDAP authentication for OnTrack and email notifications configured with an
-SMTP server.
+Since we have a functional and publicly accessible instance of Doubtfire running in GCP, the next
+stages would be to focus on the Deployment team project objectives, such as -
+
+- Create a CI/CD pipeline that automates the building, deployment, and validation of the Thoth Tech
+  OnTrack deployment onto GCP.
+- LDAP authentication for OnTrack.
+- Email notifications configured with an SMTP server.
+- Review security posture and instance sizing of the Thoth Tech OnTrack deployment in GCP.
+
+Here's a high-level diagram of using CI/CD pipeline to automate the deployment of OnTrack onto GCP -
 
 ![Google Cloud - CI/CD](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_CICD.jpg "Google Cloud - CICD")
```

Comment:
**issue (non-blocking):** Not part of this PR changeset. However, I have noted this link to be broken. Can you do a follow-up PR to fix it?